README.md

```diff
@@ -97,11 +100,12 @@ We support various module formats such as:
 
 ### 📌 Project usage
 
-* Install the dependency in your **package.json** file:
+Install the dependency in your **package.json** file:
```

Comment:
I suggest we remove the ref to "package.json". It is also retrieve the npm package and update node_modules
```suggestion
Install `bpmn-visualization` in the project:
```